src/UKHO.S100PermitService.Common/Encryption/S100Crypt.cs

```diff
@@ -20,7 +28,7 @@ public IEnumerable<ProductKey> GetDecryptedKeysFromProductKeys(IEnumerable<Produ
             _logger.LogInformation(EventIds.GetDecryptedKeysFromProductKeysStarted.ToEventId(), "Get decrypted keys from product keys started.");
 
             List<ProductKey> productKeys = [];
```

Comment:
Used new List<ProductKey>() instead of [] for list initialization.

```suggestion
            var productKeys = new List<ProductKey>();
```

Reply:
Done